candi/version_map.yml

```diff
@@ -87,6 +87,7 @@ k8s:
     cniVersion: 1.2.0
     bashible: &bashible_k8s_ge_1_23
       !!merge <<: *bashible
+    clusterautoscaler: v1.23.4
```

Comment:
i think it will be clusterAutoscaler

Reply:
something like
```
clusterAutoscalerPatch: 3
```
because we have kubernetes version in the yaml key.

Reply:
Done